packages/flame/lib/src/text/text_paint.dart

```diff
@@ -63,6 +47,7 @@ class TextPaint extends TextRenderer {
   /// However, you probably want to use the [render] method which already
   /// takes the anchor into consideration.
   /// That way, you don't need to perform the math for that yourself.
+  @Deprecated('This method will be removed in v1.4.0')
```

Comment:
Why will this be removed?

Reply:
You're right, we should postpone such decisions until after the rest of the API is in place. I've removed the deprecation note.